pkg/release/channel.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package release
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/okteto/okteto/pkg/config"
+)
+
+const downloadsUrl = "https://downloads.okteto.com"
+
+const releaseChannelDev = "dev"
+const releaseChannelBeta = "beta"
+const releaseChannelStable = "stable"
+
+// GetReleaseChannel returns the release channel for the current installation.
+// It does so by reading the $HOME/.okteto/channel file
+// If the release channel is empty it is assumed that we are on the stable channel
+func GetReleaseChannel() (string, error) {
+	filename := fmt.Sprintf("%s/channel", config.GetOktetoHome())
+	b, err := os.ReadFile(filename)
+	if err != nil && !os.IsNotExist(err) {
+		return "", err
```

Comment:
When does this file get created? Does it already exist? I want to make sure we aren't throwing errors to users that have an older cli version without this logic.

Reply:
The file may or may not exists. If the file is not there this will work as well. The error is returned for unknown errors. The caller can decide what to do with the error, I don't wanna assume here that defaulting to stable is safe. Having said this, we are returning an error to the user in the upper function. Will fix (in the other PR)